go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/felixge/httpsnoop v1.0.4
 	github.com/ipfs-shipyard/nopfs v0.0.14
 	github.com/ipfs-shipyard/nopfs/ipfs v0.25.0
-	github.com/ipfs/boxo v0.29.0
+	github.com/ipfs/boxo v0.29.2-0.20250401090827-fcc26de2640b
```

Comment:
this isn't an actual release. Do we do normally do this?

Reply:
Sometimes I think